rclcpp/include/rclcpp/node.hpp

```diff
@@ -1593,6 +1593,11 @@ class Node : public std::enable_shared_from_this<Node>
   const rclcpp::NodeOptions node_options_;
   const std::string sub_namespace_;
   const std::string effective_namespace_;
+
+  class NodeImpl;
+  // This member is meant to be a place to backport features into stable distributions,
+  // and new features targeting Rolling should not use this.  See the comment in node.cpp for more information.
```

Comment:
I suspect that this line, as-is, is going to violate our 100 character length limit.